packages/manager/modules/pci/src/projects/project/private-registry/components/plan-upgrade/template.html

```diff
@@ -0,0 +1,52 @@
+<p data-translate="private_registry_upgrade_plan_description"></p>
+<div class="row mb-4">
+    <oui-select-picker
+        class="col-md-4"
+        data-ng-repeat="plan in $ctrl.detailedPlans track by plan.id"
+        name="plan"
+        data-model="$ctrl.selectedPlan"
+        data-label="{{ plan.name }}"
+        data-variant="light"
+        data-on-change="$ctrl.onSelected(modelValue)"
+        data-values="[plan]"
+    >
+        <oui-select-picker-section>
+            <ul class="list-unstyled">
+                <li
+                    data-ng-bind="plan.registryLimits.imageStorage | cucBytes"
+                ></li>
+                <li
+                    data-translate="private_registry_upgrade_plan_connections"
+                    data-translate-values="{
+                        total: plan.registryLimits.parallelRequest,
+                        concurrency: (plan.registryLimits.parallelRequest / $ctrl.CONNECTION_CONSTANT),
+                        connectionsNumber: $ctrl.CONNECTION_CONSTANT
+                    }"
+                ></li>
+                <li
+                    data-translate="private_registry_upgrade_core_registry_99"
+                    data-ng-if="plan.name === $ctrl.PLAN_CONSTANT.SMALL"
```

Comment:
Could this be handled by the `Plan` model instead ? Something like 
```suggestion
       get cores() {
         return this.name === 'SMALL' ? 99 : 95
        }
```